src/cheri/app-cheri-instructions.adoc

```diff
@@ -285,28 +302,28 @@ include::insns/zcmp_cmva01s.adoc[]
 [#Zcmt,reftext="Zcmt"]
 === "Zcmt" Standard Extension For Code-Size Reduction
 
-The table jump instructions (<<CM.JT>>, <<CM.JALT>>) defined in cite:[riscv-code-size-spec] are _not_ redefined in {cheri_cap_mode_name} to have capabilities in the jump table. This is to prevent the code-size growth caused by doubling the size of the jump table.
+The table jump instructions (<<CM_JT_CHERI>>, <<CM_JALT_CHERI>>) defined in cite:[riscv-code-size-spec] are _not_ redefined in {cheri_cap_mode_name} to have capabilities in the jump table. This is to prevent the code-size growth caused by doubling the size of the jump table.
 
 In the future, new jump table modes or new encodings can be added to have capabilities in the jump table.
 
-The jump vector table CSR <<jvt>> has a capability alias <<jvtc>> so that it can only be configured to point to accessible memory. All accesses to the jump table are checked against <<jvtc>> in {cheri_cap_mode_name}, and against <<pcc>> bounds in {cheri_int_mode_name}. This allows the jump table to be accessed
+The jump vector table CSR <<jvt_cheri>> has a capability alias <<jvtc>> so that it can only be configured to point to accessible memory. All accesses to the jump table are checked against <<jvtc>> in {cheri_cap_mode_name}, and against <<pcc>> bounds in {cheri_int_mode_name}. This allows the jump table to be accessed
 when the <<pcc>> bounds are set narrowly to the local function only in {cheri_cap_mode_name}.
 
 NOTE: Zcmt defines that the fetch from the jump table is from instruction memory.
- The overall instruction executed is effectively 48-bit, with 16-bits from <<CM.JALT>>/<<CM.JT>>, the other 32-bits (for RV32) from the table.
+ The overall instruction executed is effectively 48-bit, with 16-bits from <<CM_JALT_CHERI>>/<<CM_JT_CHERI>>, the other 32-bits (for RV32) from the table.
```

Comment:
Tangential, but...  I'm really not sure that's a useful phrasing, even if it is in some sense true.  In at least one reading, it suggests that the table is somehow involved in compositing a 48-bit _instruction_.  Moreover, the 16 bits of the `cm.jt` or `cm.jalt` proper come from within the bounds of `pcc` and the other 32 bits come from within the bounds of `jvtc`.

Reply:
I've tried to improve the comment in the spec